src/components/display/components/StickyToggleButtonComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import { StickyToggleButtonArgs } from 'display-api'
+import { ButtonComponent } from './ButtonComponent'
+
+export const StickyToggleButtonComponent = ({ button, name, on_change, default_value }: StickyToggleButtonArgs) => {
+	const [isEnabled, setEnabled] = useState<boolean>(default_value)
+
+	const onChange = () => {
+		setEnabled(!isEnabled)
+	}
+
+	return (
+		<ButtonComponent
+			text={name ? name : button.text}
```

Comment:
This should just be the button text, nothing to do with the `name` prop (that's the variable name, e.g `DATA134`)